### ISSUE TITLE
[BUG] crc commands backed  by daemon will fail on systems with no systemd user instance support

### ISSUE THREAD

**Issue by adrianriobo** [crc-org/crc#3405]
crc daemon on linux platforms is installed based on systemd user instance, as so there is a [requirement to support it](https://github.com/crc-org/crc/blob/5847f996e4b03285a59f0496379d96cc7cc435d5/pkg/crc/preflight/preflight_linux.go#L130) in order to `crc setup` to handle the `crc daemon` installation.

If the daemon can not be installed any command backed only by the daemon will not work:

As an example `RHEL 7.9` does not support it, and current implementation for `crc status` requires it...as so on a normal flow we can:

1. `crc setup`
2. `crc start`
3. `crc status`

#### Expected 

Get the state of the cluster

#### Actual

Get an error complaining about `crc setup` not run properly

**Comment by cfergeau** [comment]
Now that RHEL9 has been released, do we want to keep supporting RHEL7? Using `oc` with RHEL7 has been deprecated with OpenShift 4.10 ( https://github.com/openshift/openshift-docs/pull/47959 ), in my opinion it's time we stop supporting it.

**Comment by adrianriobo** [comment]
Yeah that is one part of the discussion, also not aware which other edge cases for systems not supporting systemd user instance we can face @praveenkumar mention `ubuntu`?...although we do not officially support it

**Comment by praveenkumar** [comment]
@adrianriobo yes `ubuntu` is not officially supported and in our support matrix we should remove 7.x now since latest 2 ( 8 and 9) should be good enough.
